packages/cli/rollup.config.js

```diff
@@ -0,0 +1,41 @@
+import esbuild from 'rollup-plugin-esbuild'
+import resolve from '@rollup/plugin-node-resolve'
+import commonjs from '@rollup/plugin-commonjs'
+import json from '@rollup/plugin-json'
+import pkg from './package.json'
+
+const entries = [
+  'src/index.ts'
+]
+
+const external = [
+  ...Object.keys(pkg.dependencies),
+  ...Object.keys(pkg.peerDependencies),
+]
+
+export default ({ watch }) => [
+  {
+      input: entries,
+      output: {
+        dir: 'dist',
+        format: 'esm',
+        sourcemap: 'inline',
+      },
+      external,
```

Comment:
We need to tell Rollup which dependencies are external so that they are not included in our bundled Javascript. Because they are dependencies in the `package.json`, we want to let the package manager pull them under `node_modules`.